_posts/2025-08-26-strimzi-metrics-reporter.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: post
+title: "Using Strimzi Metrics Reporter to Expose Kafka Metrics in Prometheus Format"
+date: 2025-10-01
+author: Owen Corrigan
+---
+When deploying new Kafka cluster it may be one of your intentions to monitor the cluster using metrics.
```

Comment:
Not sure about this sentence. It shouldn't be  "your intentions" to monitor a cluster, it's a must do. I wonder if we can drop that sentence as below you explain the important role of metrics.